bdd/step-definitions/e2e/host-steps.ts

```diff
@@ -221,10 +221,16 @@ When("wait for {string} ms", async (timeoutMs: number) => {
 });
 
 When("sequence {string} loaded", { timeout: 50000 }, async function(this: CustomWorld, packagePath: string) {
```

Comment:
two similar steps, maybe it is good idea to remove it?

Reply:
To be honest, I really would like not deal with them in this PR. I've tried untangling step definitions in the past and it usually turned out that they are interconnected in some weird ways - changing one requires changing some global variables, which affect other steps, and so on...

So, while it seems trivial, I'd say there's a 40% chance that it will eat up half a day.

Reply:
I agree with @jan-warchol , I also tried once to unify similar steps and failed, but sooner or later it will have to be done I guess.